.env

```diff
@@ -190,3 +190,7 @@ MF_POSTGRES_READER_DB_SSL_MODE=disable
 MF_POSTGRES_READER_DB_SSL_CERT=""
 MF_POSTGRES_READER_DB_SSL_KEY=""
 MF_POSTGRES_READER_DB_SSL_ROOT_CERT=""
+
+### Twins
+MF_TWINS_HTTP_PORT=9021
+MF_TWINS_MQTT_URL=tcp://mqtt-adapter:${MF_MQTT_ADAPTER_PORT}
```

Comment:
looking at other sections in the .env file, all the possible variables are listed
here we miss the following:
MF_TWINS_LOG_LEVEL=debug
MF_TWINS_SERVER_CERT=""
MF_TWINS_SERVER_KEY=""
MF_TWINS_DB_NAME=mainflux
MF_TWINS_DB_HOST=twins-db
MF_TWINS_DB_PORT=27017
MF_TWINS_SINGLE_USER_EMAIL=""
MF_TWINS_SINGLE_USER_TOKEN=""
MF_TWINS_CLIENT_TLS=""
MF_TWINS_CA_CERTS=""